networktable.c

```diff
@@ -38,86 +38,56 @@ bool create_network_table(void)
 	return true;
 }
 
+// Private macro
+#define SQL(sql) {\
+    if(!dbquery(sql)) {\
+        logg("create_network_addresses_table(): \"%s\" failed!", sql);\
```

Comment:
Can this be generalized to all methods? Perhaps by replacing `create_network_addresses_table` with `__FUNCTION__`